src/utils/featheryClient/integrationClient.ts

```diff
@@ -53,6 +55,8 @@ export default class IntegrationClient {
   bypassCDN: boolean;
   submitQueue: Promise<any>;
   offlineRequestHandler: OfflineRequestHandler;
+  private FLINKS_TIMEOUT_MS = 40 * 1000;
```

Comment:
IMO these should be defined in the flinks file, the client files already have a lot going on